apps/accounts/verify.py

```diff
@@ -9,17 +11,27 @@
 def fxa_identify(code, config=None):
     """Get an FxA profile for an access token. If identification fails an
     IdentificationError is raised."""
-    token = get_fxa_token(code, config)['access_token']
-    return get_fxa_profile(token, config)
+    statsd.incr('accounts.fxa.identify.count.all')
+    try:
+        with statsd.timer('accounts.fxa.identify.time.all'):
+            token = get_fxa_token(code, config)['access_token']
+            profile = get_fxa_profile(token, config)
+    except:
+        statsd.incr('accounts.fxa.identify.count.fail')
```

Comment:
I thought about that but I figured this way we know that the counts are right and there isn't some code path that is being missed.
 
Although since it is in a `try`/`except`/`else` I'm actually fairly confident we won't miss anything...